cts/pacemaker-cts-dummyd.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Dummy daemon for Pacemaker CTS testing
+
+[Service]
+Type=notify
+ExecStart=@CRM_DAEMON_DIR@/pacemaker-cts-dummyd %i
+
+[Install]
+DefaultInstance=0
```

Comment:
Replace %i with 10, and drop the [Install] section

For 2.0, those allow the unit file to be installed as pacemaker-cts-dummyd@, so different callers (CTS and cts-exec) can run the daemon with different delays. For 1.1, this will only be used by CTS, so we can simplify by hard-coding the CTS delay.